homework5.js

```diff
@@ -0,0 +1,58 @@
+$('document').ready(function() {
+	console.log('this is running');
+});
```

Comment:
Your document ready function is closed way too early! It should wrap around all of your code. Right now, you are getting an error and none of your code is working because of this.